### PR TITLE
The Great JSON Escape

### PR DESCRIPTION
**Pull request by 0xkarmacoma**
We now escape the `name` and `description` from an edition when we encode them in json+base64

This fixes the issue where an edition name/description contained reserved characters (most commonly `"`)

### CODE REVIEW

**Comment by Vectorized** [comment]
Wow. LGTM.

File: contracts/EditionMetadataRenderer.sol [hunk 1]
```diff
@@ -99,14 +100,14 @@ contract EditionMetadataRenderer is EditionMetadataState {
...
-                    '", "description": "',
-                    description,
+                    '{"name":"',
+                    LibString.escapeJSON(name),
+                    '","description":"',
+                    LibString.escapeJSON(description),
                     // this is for opensea since they don't respect ERC2981 right now
-                    '", "seller_fee_basis_points": ',
+                    '","seller_fee_basis_points":',
                     StringsUpgradeable.toString(royaltyBPS),
```
Comment (Vectorized) [inline contracts/EditionMetadataRenderer.sol]: Can use `LibString.toString` instead.

File: contracts/EditionMetadataRenderer.sol [hunk 2]
```diff
@@ -99,14 +100,14 @@ contract EditionMetadataRenderer is EditionMetadataState {
...
-                    ', "fee_recipient": "',
+                    ',"fee_recipient":"',
                     StringsUpgradeable.toHexString(royaltyRecipient),
```
Comment (Vectorized) [inline contracts/EditionMetadataRenderer.sol]: Can use `LibString.toHexString` instead.

Comment (0xkarmacoma) [reply contracts/EditionMetadataRenderer.sol]: in [98d663a](https://github.com/showtime-xyz/nft-editions/pull/17/commits/98d663a712e3fdca82602b79cf5b0e35a22b3168)

File: contracts/EditionMetadataRenderer.sol (hunk 1 above)
Comment (0xkarmacoma) [reply contracts/EditionMetadataRenderer.sol]: in [98d663a](https://github.com/showtime-xyz/nft-editions/pull/17/commits/98d663a712e3fdca82602b79cf5b0e35a22b3168)

**Pull request merged**